page/sass/commit_funds.scss

```diff
@@ -47,13 +47,14 @@
       }
       .section-seperator{
         border-bottom: 1px $gray-extra-light solid;
-        width: 407px;
+        width: 90%;
         margin-left: 50px;
       }
       li{
         margin-left: 20px;
```

Comment:
This number exists already in the variable.scss file, can you use it?

Reply:
This will be taken off